native-engine/blaze/Cargo.toml

```diff
@@ -8,7 +8,7 @@ resolver = "1"
 crate-type = ["cdylib"]
 
 [features]
-default = ["tokio/rt-multi-thread", "jemalloc"]
+default = ["tokio/rt-multi-thread", "jemalloc", "poem", "serde"]
```

Comment:
They appear to be only required by the jemalloc-pprof feature, so we can add them to the jemalloc-pprof dependency. Like:

```
jemalloc-pprof = ["dep:tikv-jemalloc-ctl", "dep:tikv-jemalloc-sys", "dep:jemalloc_pprof", "dep:pprof", "dep:poem", "dep:serde", "http-service"]
```

Reply:
There's an `unused import` warning. Can you help fix it?

```
warning: unused import: `IntoResponse`
  --> native-engine/blaze/src/http/memory_profiling.rs:18:12
   |
18 | use poem::{IntoResponse, Request, RouteMethod, http::StatusCode};
   |            ^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default
```

Reply:
done